### PR TITLE
feat(helm): added servicemonitor

### LINKED ISSUE #17: Prometheus ServiceMonitor
As the webhook exposes prometheus metrics, it would be awesome if helm chart could deploy ServiceMonitor resource, so prometheus operator could autodiscover webhook scaping endpoints. https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md

### PR DESCRIPTION
**Pull request by freed-git**
fix #17 
Added service monitor to helm chart

### CODE REVIEW

**Comment by Nepoxx** [comment]
LGTM

Definitely would like this :)

**Comment by z0rc** [comment]
PR wasn't properly tested. Setting `metrics.serviceMonitor.enabled=true` fails template rendering with:
```
harbor-container-webhook/templates/servicemonitor.yaml:8:18: executing "harbor-container-webhook/templates/servicemonitor.yaml" at <.Values.serviceMonitor.labels>: nil pointer evaluating interface {}.labels
```

File: deploy/charts/harbor-container-webhook/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{{- if .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "harbor-container-webhook.fullname" . }}
+  labels:
+    {{- include "harbor-container-webhook.labels" . | nindent 4 }}
+    {{- if .Values.serviceMonitor.labels }}
```
Comment (z0rc) [inline deploy/charts/harbor-container-webhook/templates/servicemonitor.yaml:8]: Should be `{{- if .Values.metrics.serviceMonitor.labels }}`

File: deploy/charts/harbor-container-webhook/templates/servicemonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    {{- toYaml .Values.serviceMonitor.labels | nindent 4 }}
```
Comment (z0rc) [inline deploy/charts/harbor-container-webhook/templates/servicemonitor.yaml:9]: Should be `{{- toYaml .Values.metrics.serviceMonitor.labels | nindent 4 }}`

**Comment by cnmcavoy** [comment]
> PR wasn't properly tested. Setting `metrics.serviceMonitor.enabled=true` fails template rendering with:
> 
> ```
> harbor-container-webhook/templates/servicemonitor.yaml:8:18: executing "harbor-container-webhook/templates/servicemonitor.yaml" at <.Values.serviceMonitor.labels>: nil pointer evaluating interface {}.labels
> ```

Thanks for detecting this! I pushed up 0.6.2 which should correct the template issues.

**Pull request merged**